devices/Ft6xx6x/samples/Program.cs

```diff
@@ -45,11 +46,66 @@ void TouchInterrupCallback(object sender, PinValueChangedEventArgs pinValueChang
     {
         var point = sensor.GetPoint(true);
         Debug.WriteLine($"ID: {point.TouchId}, X: {point.X}, Y: {point.Y}, Weight: {point.Weigth}, Misc: {point.Miscelaneous}, Event: {point.Event}");
+        DiscrimitateButtons(point.X, point.Y);
     }
     else if (points == 2)
     {
         var dp = sensor.GetDoublePoints();
         Debug.WriteLine($"ID: {dp.Point1.TouchId}, X: {dp.Point1.X}, Y: {dp.Point1.Y}, Weight: {dp.Point1.Weigth}, Misc: {dp.Point1.Miscelaneous}, Event: {dp.Point1.Event}");
         Debug.WriteLine($"ID: {dp.Point2.TouchId}, X: {dp.Point2.X}, Y: {dp.Point2.Y}, Weight: {dp.Point2.Weigth}, Misc: {dp.Point2.Miscelaneous}, Event: {dp.Point1.Event}");
     }
-}
\ No newline at end of file
+}
+
+void DiscrimitateButtons(int x, int y)
```

Comment:
use 'ctrl + k + d' in Visual Studio, this will help in the indentation of the code. In this case, the bracket is not at the right place, the spaces is not the correct number. It may be seen as cosmetic but it drastically help reading the code.

Reply:
I cut and paste the whole Program.cs ; stitching pieces of it manually makes it impossible to also respect the indentation and will not help the newbies that don't have the whole program to start with.